daphne_worker_test/tests/test_runner.rs

```diff
@@ -237,7 +237,7 @@ impl TestRunner {
         );
 
         let gen_config = || {
-            HpkeReceiverConfig::gen(random(), HpkeKemId::X25519HkdfSha256)
+            HpkeReceiverConfig::gen(0, HpkeKemId::X25519HkdfSha256)
```

Comment:
Not strictly required, but as @bhalleycf points out, it's more realistic that we would use a counter to choose the config ID.